src/aleph/web/controllers/__init__.py

```diff
@@ -1,13 +1,62 @@
+from aiohttp import web
+import pkg_resources
+
 from aleph.web.controllers import (
-    main,
+    aggregates,
+    channels,
+    info,
     ipfs,
-    storage,
-    p2p,
+    main,
     messages,
-    aggregates,
+    p2p,
     posts,
     stats,
-    channels,
-    listener,
-    info,
+    storage,
+    version,
 )
+
+
+def register_routes(app: web.Application):
```

Comment:
+1 to centralize all routes. Should we also move them to a new file `src/aleph/web/routes.py` to make it easier to find ?

Reply:
Ok for me.